package.json

```diff
@@ -24,7 +24,9 @@
   },
   "devDependencies": {
     "copy-webpack-plugin": "4.5.1",
+    "eslint": "4.19.1",
     "git-directory-deploy": "1.5.1",
+    "prettier": "1.12.1",
```

Comment:
I think prettier is not needed

Reply:
I think if we put a `prettierrc.json` we should keep `prettier` as a dev-dependency, shouldn't we?